python/taichi/main.py

```diff
@@ -64,8 +61,10 @@ def test_cpp(args):
     # Cpp tests use the legacy non LLVM backend
     ti.reset()
     print("Running C++ tests...")
-    task = ti.Task('test')
+    task = ti.Task('test_gui')
     return int(task.run(*test_files))
```

Comment:
I think I found the correct usage?

Reply:
Sorry, but `test_gui` was for testing the GUI system.